biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -659,6 +658,9 @@ void doPlugins() {
 				e.printStackTrace(System.err);
 				error("Analyzer Plugin %s failed %s", plugin, e);
 			}
+			finally {
+				endHandleErrors(previous);
```

Comment:
This is wrong. Now the error in the exception block will be put in the wrong Processor. You need to wrap the try/finally just around the call to plugin.analyzeJar.